src/options.rs

```diff
@@ -383,6 +386,13 @@ impl Options {
         self
     }
 
+    /// Select option to include retry logic with backoff 
```

Comment:
Would be good to update the docs and examples so they are on the same level as other options.
Also: naming without `with` for consistency.